client/components/dataviews/style.scss

```diff
@@ -1,15 +1,19 @@
 @import "@wordpress/base-styles/breakpoints";
 @import "@wordpress/base-styles/mixins";
+@import "@wordpress/dataviews/build-style/style.css";
 
 .dataviews-wrapper {
+	// Maybe move upstream to the gutenberg repository
+	width: 100%;
```

Comment:
Without this, it seems the dataviews take 0 width. Maybe this style should be moved directly to the dataviews package in the Gutenberg repository.